platform/osx/src/MGLMapView.mm

```diff
@@ -375,7 +375,6 @@ - (void)installAttributionView {
 
 /// Adds gesture recognizers for manipulating the viewport and selecting annotations.
 - (void)installGestureRecognizers {
-    self.acceptsTouchEvents = YES;
```

Comment:
This breaks touch events, right? Even if we conditionalize it on `MGLScrollWheelZoomsMapViewDefaultKey`, gestures like rotation and two-finger tap will be broken.